bite-size-notes/week-8-a-Probability Review-Zhijie Tan

```diff
@@ -0,0 +1,204 @@
+
+
+### Discrete Random Variable Plot
+
+The following plot represents the probability mass function (PMF) of a discrete random variable: 
+![Discrete Random Variable Plot](PMF.png)
+- $S=[1.2.3]$
+- P($x\in$,$\mathcal{S}$)= $\sum_{x=S}P(x)$
+- $\sum_xP(x)=1$ 
+- $0≤P（x）≤1$
+
+### Continous Random Variable Plot
+The following plot represents the probability density function (PDF) of a continuous random variable:
+
+![Discrete Random Variable Plot](PDF.png)
+
+- P($x\in$,$\mathcal{S}$)= $\int_\mathcal{S}P(x) dx$
+- $\int_x P(X)=1$
+  
+### The probability density function of a multivariate normal (or Gaussian) distribution
+$N(u, \Sigma) = \frac{1}{(2\pi)^{k/2} \det(\Sigma)^{1/2}} \exp\left(-\frac{1}{2}(x - u)^T \Sigma^{-1} (x - u)\right)$
+
+- $u$: the mean vector
+- $\Sigma$:the covariance matrix
+- $k$:the dimensionality of the distribution
+- $x$:the vector of the variables
+- $det(\Sigma)$:the determinant of the the covariance matrix 
+- $\Sigma^{-1}$is the inverse of the covariance matrix
+
+## 
+$\mathbf{X} = (x_1, x_2, \ldots, x_n)$, if $\mathbf{X}$ is a vector of random variables
+
+$P(\mathbf{X})=P(x_1, x_2, \ldots, x_n)$
+
+$(\mathbf{X,Y}) \sim P(\mathbf{X,Y})$
+
+$((x_1, X_2, \ldots, x_n),(y_1, y_2, \ldots, y_n))\sim P((x_1, x_2, \ldots, x_n,y_1, y_2, \ldots, y_n))$
+
+## Marginalization
+Given $(X,Y) \sim P(X,Y)$ ,what is $P(X), P(Y)$
+- “Integrate out" a variable
+- $[ P(X) = \int_y P(X, Y) \, dy / {\Sigma_y} P(X, Y) ]$
+- $[ P(Y) = \int_x P(X, Y) \, dx / {\Sigma_x} P(X, Y) ]$
+
+## Conditional distribution 
+Given $X \& Y$ ,what is $X \mid Y$ $(X$ given $Y)$ 
+
+$P(X \mid Y) = \frac{P(X,Y)}{P(Y)}$
+
+
+## Chain rule
+$P(X,Y)=P(X|Y)P(Y)=P(Y|X)P(X)$
+
+$P(X,Y,Z)=P(X|Y,Z)P(Y,Z)=P(X|Y,Z)P(Y|Z)P(Z)$
+
+## Bayes‘ Rule
+$P(X|Y)P(Y)=P(Y|X)P(X)$
+
+$P(X|Y)=\frac{P(Y|X)P(X)}{P(Y)} \int_x P(Y|X)P(X) dx$
+
+$P(X|Y) \space \propto \space P(Y|X)P(X)$ ($\propto$: is proportional to)
+
+## Posterior, if Y is a measurement then what is X given Y?
+## Prior, your belief guess of X before knowing Y?
+### Likelihood Function
+ The joint probability mass (or probability density) of observed data is viewed as a function of the parameters of a statistical model.[2]
+- Given X, what you expect Y to be
+- What happens if we have multiple measures?
+- $P(x|y,y_2) \space \propto \space P(y_2|x,y_1)P(x|y_1)$
+
+Assume measurements are independent
+- - $\propto \space P(y_2|x)P(x|y_1)$
+- - $\propto \space P(y_2|x,y_1)P(y_2|x)$   
+- - $\propto \space P(y_2|x,y_1)P(y_1|x)P(x)$ 
+
+## Recursive Bayes Estimates
+A general probabilistic approach for recursively estimating an unknown probability density function (PDF) over time using incoming measurements and a mathematical process model.[2]
+
+Recursive Bayes Estimation Example:
+-- Python Coding
+-- [Recursive Bayes Estimation](https://github.com/ankit-1517/Recursive-Bayesian-Estimation/blob/main/bayes_filter.ipynb)[3]
+
+
+$x$ is (static)
+
+$P(x|y_{1：k})=P(y_k|x)P(x|y_{1:k-1})=\int_x P(y_k|x)P(x|y_{1:k-1})dx$
```

Comment:
This formula shows how the posterior at step k becomes the prior for step k + 1.

Reply:
Fixed. Thank you!